packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/messages/CollaborativeMessageService.java

```diff
@@ -34,7 +34,12 @@ public CollaborativeMessageService(@Qualifier("collaborativeMessageSourceAccesso
 
     @Override
     public String invalidInput(String expectedInputTypeName, String receivedInputTypeName) {
-        return this.messageSourceAccessor.getMessage("INVALID_INPUT", new Object[] { expectedInputTypeName, receivedInputTypeName });
+        return this.messageSourceAccessor.getMessage("INVALID_INPUT", new Object[] {expectedInputTypeName, receivedInputTypeName});
+    }
+
+    @Override
```

Comment:
Nothing in `sirius-components-collaborative` uses this new message, I suspect that it does not belong there.